tests/common/snappi_tests/common_helpers.py

```diff
@@ -1029,3 +1037,118 @@ def start_pfcwd_fwd(duthost, asic_value=None):
         stop_pfcwd(duthost, asic_value)
         duthost.shell('sudo ip netns exec {} pfcwd start --action forward 200 --restoration-time 200'.
                       format(asic_value))
+
+
+def clear_counters(duthost, port):
+    """
+    Clear PFC, Queuecounters, Drop and generic counters from SONiC CLI.
+    Args:
+        duthost (Ansible host instance): Device under test
+        port (str): port name
+    Returns:
+        None
+    """
+    if (duthost.is_multi_asic):
+        asic = duthost.get_port_asic_instance(port).get_asic_namespace()
+        duthost.shell("sudo ip netns exec {} sonic-clear queuecounters \n".format(asic))
+        duthost.shell("sudo ip netns exec {} sonic-clear dropcounters \n".format(asic))
+        duthost.shell("sudo sonic-clear counters \n")
+        duthost.shell("sudo sonic-clear pfccounters \n")
+    else:
+        duthost.shell("sonic-clear queuecounters \n")
+        duthost.shell("sonic-clear dropcounters \n")
+        duthost.shell("sudo sonic-clear counters \n")
+        duthost.shell("sudo sonic-clear pfccounters \n")
+
+
+def interface_stats(duthost, port):
```

Comment:
There is one parse_portstat() function, can we reuse that one?

Reply:
Thanks for the comment. Incorporated parse_portstat to fetch value for the given DUT and interface.